db/database.py

```
@@ -0,0 +1,537 @@
+    fake_pre_keys["signedPreKey"] = {
+        "publicKey": spk.public_key().to_base64(),
+        "privateKey": spk.private_key().to_base64(),
+        "keyId": spk_record.id().get_id(),
+        "signature": base64.b64encode(spk_record.signature()).decode(),
+    }
+    # data = json_join_public(fake_pre_keys["pre]"], fake_secret_pre_keys)
+
+    print(fake_pre_keys)
+    fake_pre_keys["type"] = "aci"
+    fake_pre_keys["aci"] = "bobs"
+    mitmb = MitmBundle.model_validate(fake_pre_keys)
```

Ruff

Undefined name `MitmBundle`

(F821)